src/home/home.ts

```diff
@@ -1103,22 +1125,56 @@ function onWindowResize(evt: Event): void {
     }
     if (window.innerWidth < 1090) {
         contentArea.className = 'row content sidebar-hide';
+        if (messagePane.classList.contains('pane-spacer')){
+            messagePane.classList.remove("pane-spacer");
+        }
+        defaultSidebar.hide();
+        defaultSidebar.type='Over';
     } else {
-        hideSideBar();
+        if (!isNewMailClick){
```

Comment:
Simplify this.
Refer the line: https://github.com/essential-studio/ej2-button-components/blob/development/src/speed-dial/speed-dial.ts#L899